### PR TITLE
simplify and groovify example

### PR DESCRIPTION
**Pull request by mguillem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,18 +69,18 @@ Additionally you can pass a JSONArray as a String in order to send complex
...
+node {
+    def attachments = [
+        [
+            text: 'I find your lack of faith disturbing!',
+            fallback: 'Hey, Vader seems to be mad at you.',
+            color: '#ff0000'
+        ]
+    ]
+
+    slackSend(color: '#00FF00', channel: '@gustavo.maia', attachments: JsonOutput.toJson(attachments))
```
Comment (timja) [inline README.md]: Does this require any whitelisting or it works OOTB?

Comment (mguillem) [reply README.md]: you're right, seems I had a whitelisting already configured for JsonOutput.toJson. Not optimal. Converstion to JSON should be handle in slackSend. I'll try to find time to propose a patch for it.

Comment (timja) [reply README.md]: please do, that would be great <3

**Comment by mguillem** [comment]
Closing. Won't work OOTB.

**Pull request closed without merging**